web-registry/src/pages/Roles/Roles.tsx

```diff
@@ -118,7 +122,6 @@ const Roles: React.FC = () => {
     fetchPolicy: 'cache-and-network',
   });
   const project = data?.projectById;
-  const creditClassId = project?.creditClassByCreditClassId?.onChainId;
```

Comment:
We don't rely on the db credit class onChainId, so this lookup wasn't helpful anymore.